backend/flaskr/utils.py

```diff
@@ -286,25 +312,56 @@ def get_companies_from_database(
     Returns:
             Array of dictionaries of company data
     """
-    #Construct SQL Query
```

Comment:
This change would undo the SQL injection fix. Maybe you forgot to sync the most recent changes before making edits?